src/main/webapp/about.html

```diff
@@ -19,15 +19,14 @@
   <body onload="onAboutLoad()">
     <div id="content">
       <nav class="navbar navbar-expand-lg navbar-light bg-light">
-        <span class="navbar-brand mb-0 h1">InterviewMe</span>
+        <a href= "index.html"><span class="navbar-brand mb-0 h1">InterviewMe</span></a>
```

Comment:
I think these should instead have the path "/", which will result in a cleaner URL and achieve the same effect.

Also, please remove the space after the equals sign.

Reply:
Done